kratos/tests/test_linear_master_slave_constraints.py

```diff
@@ -1,18 +1,18 @@
 from __future__ import print_function, absolute_import, division
 
 # Importing the Kratos Library
-import KratosMultiphysics.KratosUnittest as KratosUnittest
 import KratosMultiphysics
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics.kratos_utilities as KratosUtils
+
 try:
     import KratosMultiphysics.StructuralMechanicsApplication
     missing_external_dependencies = False
     missing_application = ''
 except ImportError as e:
     missing_external_dependencies = True
     # extract name of the missing application from the error message
-    import re
-    missing_application = re.search(r'''.*'KratosMultiphysics\.(.*)'.*''',
-                                    '{0}'.format(e)).group(1)
+    missing_application = KratosUtils.HandleInvalidImportException(e)
```

Comment:
@roigcarlo for this I started to use IsApplivationAvailable or AreApplicationsAvailable from kratos_utilities.
Seemed easier to me, what do you think?

Reply:
I'll take a look :+1:

Reply:
Here is an example:
https://github.com/KratosMultiphysics/Kratos/blob/master/applications/MeshMovingApplication/tests/test_ale_fluid_solver.py